src/site/markdown/api-separation.md

```diff
@@ -0,0 +1,241 @@
+<dependency>
+    <groupId>org.apache.logging.log4j</groupId>
+    <artifactId>log4j-core</artifactId>
+    <version>2.17.0</version>
+</dependency>
+```
+
+Note that having two different versions of log4j-api and log4j-core on your
+classpath is not guaranteed to work correctly(e.g. 2.15 of log4j-api and 
+2.17 of log4j-core).
+
+##  Using Log4j2 API with Logback
+
+Since the Log4j2 API is generic, we can use it to send messages via SLF4J
+and then have Logback do the actual logging of the messages.  This means
+that you can write your code tied to the Log4j2 API, but users of your
+code do not need to use the Log4j2 core if they are already using Logback.
+
+To switch to using Logback, you will need to add the following to your
+dependencies(assumging that you are using Maven):
+
+```
+<dependency>
+    <groupId>org.apache.logging.log4j</groupId>
+    <artifactId>log4j-api</artifactId>
+    <version>2.17.0</version>
+</dependency>
+<dependency>
+    <groupId>org.apache.logging.log4j</groupId>
+    <artifactId>log4j-to-slf4j</artifactId>
+    <version>2.17.0</version>
+</dependency>
+<dependency>
+      <groupId>ch.qos.logback</groupId>
+      <artifactId>logback-classic</artifactId>
+      <version>1.2.10</version>
+</dependency>
+```
+
+## Using Log4j2 as an SLF4J Implementation
+
+If you don't want to depend on the Log4j2 API and instead want to use SLF4J,
+that is possible as well.  Assuming that our code looks like the following:
+
+```java
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class Log4j2Test {
+
+    private static final Logger logger = LoggerFactory.getLogger(Log4j2Test.class);
+
+    public Log4j2Test(){
+        logger.info( "Hello World!" );
+    }
+}
+```
+
+We can then route the messages to Log4j2 using the log4j-slf4j-impl like the following:
+
+```
+<dependency>
+    <groupId>org.slf4j</groupId>
+    <artifactId>slf4j-api</artifactId>
+    <version>1.7.32</version>
+</dependency>
+<dependency>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
+      <version>2.17.0</version>
+</dependency>
+<dependency>
+    <groupId>org.apache.logging.log4j</groupId>
+    <artifactId>log4j-core</artifactId>
+    <version>2.17.0</version>
+</dependency>
+```
+
+Note that if we were using SLF4J 1.8 instead of 1.7, that requires us to use
+log4j-slf4j18-impl instead of log4j-slf4j-impl.
+
+## Using Log4j2 with JUL
+
+It is also possible to route messages that are logged using java.util.logging
+to Log4j2.  Assuming that the code looks like the following:
+
+```java
+import java.util.logging.Logger;
+
+public class Log4j2Test {
+
+    private static final Logger logger = Logger.getLogger(Log4j2Test.class.getName());
+
+    public Log4j2Test(){
+        logger.info( "Hello World!" );
+    }
+```
+
+We can then also route these messages to the Log4j2 core by adding in the JUL bridge,
+and setting the java.util.logging.manager property in the JVM(see the documentation on
```

Comment:
It might be helpful to provide the precise arg here, otherwise folks may read over it. I've seen it done incorrectly a few places.
```suggestion
and setting `-Djava.util.logging.manager=org.apache.logging.log4j.jul.LogManager` on the JVM (see the documentation on
```